src/data/markdown/docs/03 cloud/05 Project and Team Management/03 Members.md

```diff
@@ -16,11 +16,11 @@ The following roles are supported:
 
 ### Owner
 
-The organization owner has unrestricted access to the organization data and settings.
+The organization's owner has unrestricted access to the organization data, settings and all its projects and member roles. The owner can pass ownership to another member by using the _Transfer ownership_ link from the organization settings.
```

Comment:
I'm not sure if the _Transfer ownership_ is pointed out anywhere, maybe there needs to be an image of it somewhere to make it clear?